src/components/LayoutGrid.vue

```diff
@@ -66,6 +74,52 @@ export default defineComponent({
       });
     });
 
+    // Construct the common instance of vtkResliceCursorWidget and provide it
+    // to all the child ObliqueView components.
+    const resliceCursor = ref<vtkResliceCursorWidget>();
+    resliceCursor.value = vtkResliceCursorWidget.newInstance();
+    // Orient the planes of the vtkResliceCursorWidget to the orientation
+    // of the currently set image.
+    const resliceCursorState = resliceCursor.value.getWidgetState() as ResliceCursorWidgetState;
+    provide(VTKResliceCursor, resliceCursor);
+
+    watch(currentImageData, (image) => {
+    // Reset to default plane values before transforming based on current image-data.
+    resliceCursorState.setPlanes({
+      4: {
```

Comment:
I think it is clearer to use `[ViewTypes.XY_Plane]`,  `[ViewTypes.YZ_Plane]`, `[ViewTypes.XZ_Plane]` syntax instead of 4, 5, 6.